src/sqlite-xamarin/sqlite-xamarin.targets

```diff
@@ -26,10 +26,13 @@
       -->
     <_LibSqliteXamarin Include="@(_SupportedAbi->'src\main\libs\%(Identity)\libsqlite3_xamarin.so')" />
   </ItemGroup>
+  <PropertyGroup>
+	<APP_ABI>$([System.String]::Copy('$(AndroidSupportedDeviceAbis)').Replace (';', ' '))</APP_ABI>
```

Comment:
@grendello we generally don't upper case properties. If its internal just precede it with a _ 
eg. <_AppAbi>